src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -153,7 +153,7 @@ internal ExecutionContext ContextCopy
                 ExecutionContext context = _context;
                 if (context != null)
                 {
-                    return context.CreateCopy();
+                    return context; // No need to copy on CoreCLR; ExecutionContext is immutable
```

Comment:
I can confirm that this code is shared with NetNative only but not .Net Desktop. The Desktop contract is a mixed forwarding assembly + extension methods to support the new TPL APIs.